src/delimiter.ts

```diff
@@ -31,22 +30,18 @@ import {getCharacterMetrics} from "./fontMetrics";
 import symbols from "./symbols";
 import utils from "./utils";
 
-import type Options from "./Options";
-import type {CharacterMetrics} from "./fontMetrics";
-import type {HtmlDomNode, DomSpan, SvgSpan} from "./domTree";
-import type {Mode} from "./types";
-import type {StyleInterface} from "./Style";
-import type {VListElem} from "./buildCommon";
+import Options from "./Options";
+import {CharacterMetrics} from "./fontMetrics";
+import {HtmlDomNode, DomSpan, SvgSpan} from "./domTree";
+import {Mode} from "./types";
+import {StyleInterface} from "./Style";
+import {VListElem} from "./buildCommon";
```

Comment:
nit: some of the other files have spaces between the `{}` and the stuff inside them.